pkg/kubelet/cm/topologymanager/scope.go

```diff
@@ -31,6 +31,8 @@ const (
 	containerTopologyScope = "container"
 	// podTopologyScope specifies the TopologyManagerScope per pod.
 	podTopologyScope = "pod"
+	// noneTopologyScope specifies the TopologyManagerScope when topologyPolicyName is none.
+	noneTopologyScope = "none"
```

Comment:
will this be exposed or should this be kept as implementation detail? (I think the latter unless there are strong reasons to expose)

Reply:
Currently I think there is no need to Expose this.